app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -129,4 +145,20 @@ public void updateItems(String[] translationNames,
   public interface OnSelectionChangedListener {
     void onSelectionChanged(Set<String> selectedItems);
   }
+
+  public void setOnClickListener(View.OnClickListener onClickListener) {
+    this.textListener = onClickListener;
+  }
+
+  public String[] updateTranslationNames(String[] translationNames) {
+
+    List<String> transList = new ArrayList<>();
```

Comment:
Nitpick: I don't like the `trans` prefix. Can you make it `translationsList` or (since, it is a local variable) simply `list` would also work.